lib/register.coffee

```diff
@@ -19,7 +26,12 @@ exports.generateUUID = ->
 	# pass the certificate validation in OpenVPN This either means that
 	# the RFC counts a final NULL byte as part of the CN or that the
 	# OpenVPN/OpenSSL implementation has a bug.
-	return crypto.pseudoRandomBytes(31).toString('hex')
+	Promise.try ->
+		crypto.randomBytes(31).toString('hex')
+	.catch ->
```

Comment:
I don't know the specific error, the docs just say it throws *something* https://nodejs.org/api/crypto.html#crypto_crypto_randombytes_size_callback